From 9bb09886320eb286108fb370b2634a66b3e3b9ff Mon Sep 17 00:00:00 2001
From: Joe Slater <joe.slater@windriver.com>
Date: Thu, 21 Mar 2024 16:28:31 +0000
Subject: [PATCH] systemd: enable mac based names in NamePolicy

The default NamePolicy for network interface names does not
include names based on mac addresses.  Some BSPs, though, do
not provide information to compute other names, so we enable
mac names as a last resort.

Upstream-Status: Inappropriate [enable feature]

Signed-off-by: Joe Slater <joe.slater@windriver.com>
---
 network/99-default.link | 4 ++--
 1 file changed, 2 insertions(+), 2 deletions(-)

diff --git a/network/99-default.link b/network/99-default.link
index 429ac31e80..543ce54661 100644
--- a/network/99-default.link
+++ b/network/99-default.link
@@ -15,6 +15,6 @@
 OriginalName=*
 
 [Link]
-NamePolicy=keep kernel database onboard slot path
-AlternativeNamesPolicy=database onboard slot path
+NamePolicy=keep kernel database onboard slot path mac
+AlternativeNamesPolicy=database onboard slot path mac
 MACAddressPolicy=persistent
-- 
2.35.5

